src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -22,5 +24,12 @@ public class FeatureManagementOptions
         /// The default value is true.
         /// </summary>
         public bool IgnoreMissingFeatures { get; set; } = true;
+
+        /// <summary>
+        /// Controls the duration of the cache from Parameters to Settings.
```

Comment:
There a few points that I think are worth mentioning in this description. I wrote up something that I felt covered them all.

Controls the cache lifetime of settings bound by <cref=IFilterParametersBinder> in the feature management cache.
By default, this cache is off, with a ttl of <cref=TimeSpan.Zero>.
To enable caching of filter parameters, a non-zero ttl should be provided. A recommendation is five seconds.
Increasing the value may cause an observed increase in memory footprint as items live longer.
Lowering the value will decrease performance benefits yielded by caching bound parameters.